data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaSourceCustomConsumer.java

```diff
@@ -211,14 +218,14 @@ Map<TopicPartition, OffsetAndMetadata> getOffsetsToCommit() {
 
     @Override
     public void run() {
-        try {
-            consumer.subscribe(Arrays.asList(topicName));
-            while (!shutdownInProgress.get()) {
+        consumer.subscribe(Arrays.asList(topicName));
+        while (!shutdownInProgress.get()) {
+            try {
+                resetOrCommitOffsets();
                 consumeRecords();
-                commitOffsets();
+            } catch (Exception exp) {
+                LOG.error("Error while reading the records from the topic...", exp);
```

Comment:
I think we should avoid ellipsis in our logs. It's not a big deal, but it seems either indicate that we trailed of or have more to say (not the exception).